docs/contributing/installing.md

```diff
@@ -0,0 +1,52 @@
+---
+id: install
+title: Contributing
+sidebar_label: Install
+---
+
+# Install
+
+## Prerequisits
+
+### Setup a development environment
+
+1. Install the latest stable versions of [Git](https://git-scm.com/download).
```

Comment:
What if I'm using the desktop app or just VSCode? What are those requirements? I'd prefer not to maintain detailed information about git if we don't have to. We should either note that `git` is a requirement and we recommend using the CLI for some reason

Reply:
removing